### DISCUSSION TITLE
Student Discount

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#118] [Anything Else]
"It would be great if Plausible has special rate for student."

([imported from the old roadmap](https://plausible.nolt.io/66))

**Comment by yagarea** [comment]
[This organization](https://www.isic.org/) offering partnerships and API.

https://www.isic.org/

**Comment by Tragio** [comment]
Be part of Github Students would be very nice https://education.github.com/pack/offers I find great software there 😄

**Comment by metmarkosaric** [comment]
> Be part of Github Students would be very nice https://education.github.com/pack/offers I find great software there smile

Thanks! How do we join this?

**Reply by turnernat0r** [reply]
@metmarkosaric did anything come to light of this? Would be great.

**Comment by Tragio** [comment]
> > Be part of Github Students would be very nice https://education.github.com/pack/offers I find great software there smile
> 
> 
> 
> Thanks! How do we join this?

[Here](https://education.github.com/pack/partners) 🙂

**Comment by haula1122** [comment]
Please , I applied for GitHub student pack but unfortunately I was rejected for not using school issued email, and the fact is our school don't issues email to us, I have submitted my ID card as proof of my studentship... 
Thanks

**Comment by scratchmex** [comment]
What happened to this? I'm interested.

**Comment by hecker** [comment]
Hi, is there an update regarding the student discount? 😄
